lib/config_read.rb

```diff
@@ -0,0 +1,68 @@
+# frozen_string_literal: true
+
+$LOAD_PATH.push('fluentd/lib')
```

Comment:
Ideally we should not change the load_path in code. Can we make all the dependency manageable under the same directory under this repo? seems like you already include the fluentd library into your folder based on this:
> I've copied over the entire repository (not just specific files) into my project folder for now, but will research alternatives too.

@davidbtucker can you take a look on this? Looks like you added something for filter analyze config (https://github.com/GoogleCloudPlatform/fluent-plugin-google-cloud/blob/fad17b7c203a522d62503e93a27961ffb8bc035b/lib/fluent/plugin/filter_analyze_config.rb#L16-L17) 

Also is it related to dependency added as gemspec here? https://github.com/GoogleCloudPlatform/fluent-plugin-google-cloud/blob/fad17b7c203a522d62503e93a27961ffb8bc035b/fluent-plugin-google-cloud.gemspec#L22

Reply:
I figured out the issue with the dependencies - turns out you had to download fluentd as a gem, not make a local repo, which is what I was doing. Now that it is a gem, I can require the files normally, just like the page you linked above does. Thanks for including those!